prospero-common/src/main/java/org/wildfly/prospero/api/ArtifactUtils.java

```diff
@@ -31,25 +34,39 @@ public static int compareVersion(Artifact first, Artifact other) {
         return new ComparableVersion(first.getVersion()).compareTo(new ComparableVersion(other.getVersion()));
     }
 
-    public static ChannelManifestCoordinate manifestFromString(String urlGavOrPath) {
+    public static ChannelManifestCoordinate manifestCoordFromString(String urlGavOrPath) {
+        return coordinateFromString(urlGavOrPath, ChannelManifestCoordinate.class);
+    }
+
+    public static ChannelCoordinate channelCoordFromString(String urlGavOrPath) {
```

Comment:
can you add tests for both `channelCoordFromString` and `manifestCoordFromString` - want to make sure we catch if the constructors change

Reply:
Sure, test added.